frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+import random
+import string
+
+import pytest
+import shakedown
+
+## Commented out so it doesn't actually run.
+## Using the PR for comments and feedback, don't want the tests.
+#@pytest.fixture(scope='session', autouse=True)
+def configure_universe(request):
+    stub_urls = {}
+
+    # prepare needed universe repositories
+    stub_universe_urls = os.environ.get('STUB_UNIVERSE_URL')
+    if not stub_universe_urls:
+        return
+    for url in stub_universe_urls.split(' '):
+        package_name = 'testpkg-'
+        package_name += ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(8))
+        stub_urls[package_name] = url
+
+    try:
+        # clean up any duplicate repositories
```

Comment:
Good question. Again: I copied the logic from `run_tests.py`. I can only speculate as to why, will need to ask about it.